src/main/cpp/audioengine.cpp

```diff
@@ -323,16 +323,10 @@ namespace AudioEngine
             {
                 int lAmount = channel->liveEvents.size();
 
-                // the volume of the live events is divided by the channel mix as a live event
-                // is played on the same instrument, but just as a different voice (note the
-                // events can have their own mix level)
-
-                float lAmp = channelVolume > 0.f ? 1.0 / channelVolume : 1.0;
-
```

Comment:
The result of having Git access when smoking crack. Channel volume is a range between 0 - 1. At the maximum volume (1.0) a live event would have a volume (of 1/1 ==) 1.0. At half volume (0.5), a live event would have a volume of (1/0.5) == 2.0, wait ??

Reply:
Doubt a crack addict could still write C++ ;-)